subsys/net/ip/Kconfig.mgmt

```diff
@@ -23,18 +23,18 @@ config NET_MGMT_EVENT
 	(or else) needs to be notified on a specific network event
 	(ip address change for instance) to trigger some related work.
 
+if NET_MGMT_EVENT
```

Comment:
This looks like unrelated change, perhaps create another commit for this.

Reply:
ok